docs/source/quickstart.rst

```diff
@@ -180,14 +180,14 @@ Finally, we start the engine on the training dataset and run it during 100 epoch
 **Where to go next?** To understand better the concepts of the library, please read :doc:`concepts`.
 
 
-.. rubric:: Footnotes
+.. Footnotes ::
```

Comment:
Please, see  : https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#footnote

Reply:
I got it from here. It should be looking like this one [Click here](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#footnote1)